### ISSUE TITLE
error in case

### ISSUE THREAD

**Issue by b2whats** [jsstyles/jss#252]
@kof 
```js
const sheet = jss.createStyleSheet({}, options)
sheet.addRule('test', {color: red})
sheet.attach()  // componentWillMount
sheet.detach() // conmonentWillUnmount

sheet.addRule('test2', {color: blue})
```
file: **StyleSheet.js**
```js
  addRule(name, style) {
    const rule = this.createRule(name, style)
    // Don't insert rule directly if there is no stringified version yet.
    // It will be inserted all together when .attach is called.
    if (this.deployed) {
      const renderable = this.renderer.insertRule(rule)
      if (this.options.link) rule.renderable = renderable
    }
    return rule
  }
```

after sheet.detach() we delete **style** tag in DOM tree, but not change this.deployed, it equal true
and when you try to add a new style we get error

```js
  insertRule(rule) {
    // IE8 has only `styleSheet` and `styleSheet.rules`
    const sheet = this.element.sheet || this.element.styleSheet
    const cssRules = sheet.cssRules || sheet.rules  // cssRules not defined
    const nextIndex = cssRules.length
    if (sheet.insertRule) sheet.insertRule(rule.toString(), nextIndex)
    else sheet.addRule(rule.selector, rule.toString({selector: false}), nextIndex)
    return cssRules[nextIndex]
  }
```

**Comment by kof** [comment]
fixed and released
